tests/test_api_admin.py

```
@@ -1,11 +1,53 @@
+]
+
+ADMIN_ACTION_ENDPOINTS = [
+    "/admin/trigger-crawling",
+    "/admin/trigger-bus-notice",
+    "/admin/trigger-route-check",
+    "/admin/trigger-zone-check",
+    "/admin/trigger-test-alarm-for-user",
+]
+
+ADMIN_USER = "admin"
+ADMIN_PASS = "secret123"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "ADMIN_PASS"

(S105)